projects/plugins/jetpack/_inc/client/traffic/ads.jsx

```diff
@@ -375,7 +358,7 @@ export const Ads = withModuleSettingsFormHelpers(
 								/>
 								<span className="jp-form-setting-explanation">
 									{ __(
-										'Adds a link to your privacy policy to the bottom of the CCPA notice popup (optional).',
+										'Adds a link to your privacy policy to the bottom of the sale opt-out notice popup (optional).',
```

Comment:
Did you mean "state" instead of "sale" on this line?

Reply:
I meant "sale". The word "sale" seems to be consistently used in the 3 documents provided by legal to be used across the settings page and the "do not sell" modal.
I assumed it relates to the fact that personal information is considered being "sold" to advertisers, and the "do not sell" link copy says Do Not Sell or Share My Personal Information.

Reply:
On 2022-11-08, the copy was updated to the following in the source GDoc. Please implement this change accordingly:

```suggestion
										'Adds a link to your privacy policy to the notice popup triggered by the do not sell link (optional).',
```

Reply:
I wasn't tagged and the line on the Google Docs wasn't highlighted as changed. I have now made the change, thanks for bringing it up.